Lib/idlelib/idle_test/run_coverage.py

```diff
@@ -0,0 +1,168 @@
+import argparse
+import contextlib
+import os
+import pathlib
+import subprocess
+import sys
+import textwrap
+
+
+def main():
+    idlelib_path = find_idlelib_path()
+    sources_root_path = idlelib_path.parent.parent
+    venv_path, venv_python_path = ensure_venv(sources_root_path)
```

Comment:
```suggestion
    venv_path, venv_python_path = ensure_venv(idlelib_path)
```
I think it makes more sense to pass the idlelib path as we are immediately joining "Lib/idlelib" in the method.

Reply:
The sources root path is now also used to find the local Python executable, so I'm leaving it as-is.